src/jloptions.c

```diff
@@ -109,6 +111,10 @@ static const char opts[]  =
     " --color={yes|no}          Enable or disable color text\n"
     " --history-file={yes|no}   Load or save history\n\n"
 
+    // error and warning options
+    " --depwarn={yes|no|error}  Enable or disable syntax and method deprecation warnings (\"error\" turns warnings into errors)\n"
```

Comment:
Should we name  this `--warn-deprecations`? (doesn't need to be the same PR though)